src/grid-list/grid-list.jsx

```diff
@@ -105,16 +102,16 @@ const GridList = React.createClass({
       }
       const childCols = currentChild.props.cols || 1;
       const childRows = currentChild.props.rows || 1;
-      const itemStyle = this.mergeStyles(styles.item, {
+      const itemStyle = Object.assign({}, styles.item, {
```

Comment:
Good point, it wasn't necessary here. My faulty logic when I did this was because it's inside a loop of children and I didn't want `styles.item` to be mutated and contain data from other items so I decided to  create an new object this time. But in this case the keys never change so they should always be overwritten so it should be safe.